src/NoUnused/Patterns.elm

```diff
@@ -403,6 +403,11 @@ pluralDetails =
     [ "You should either use these values somewhere, or remove them at the location I pointed at." ]
 
 
+redundantDetails : List String
+redundantDetails =
+    [ "This pattern is redundant and should be replaced with '_' at the location I pointed at." ]
```

Comment:
I feel like "this pattern" and "at the location I pointed at" is (also) redundant.
Do you think "This pattern is redundant and should be replaced with '_'." would be enough?

It would be consistent with the other error details but I'm not sure that is very important for a user in this case.

Reply:
I think we can remove "at the location I pointed at" I only left it in because that's what you/we wrote the first time around 👍

Reply:
Yeah let's remove it then. Old Jeroen had good ideas but this one wasn't part of those.